includes/block-patterns/course-list/class-sensei-course-list-block-patterns.php

```diff
@@ -43,43 +43,50 @@ public function register_course_list_block_patterns() {
 				'content'     => '<!-- wp:query {"query":{"offset":0,"postType":"course","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":"","perPage":6},"displayLayout":{"type":"list"},"layout":{"inherit":true}} -->
 					<div class="wp-block-query wp-block-sensei-lms-course-list wp-block-sensei-lms-course-list--is-list-view">
 						<!-- wp:post-template -->
-							<!-- wp:post-featured-image {"isLink":true,"align":"center"} /-->
-							<!-- wp:columns -->
-								<div class="wp-block-columns">
 
-									<!-- wp:column {"width":"66.66%"} -->
-										<div class="wp-block-column" style="flex-basis:66.66%">
-											<!-- wp:sensei-lms/course-categories /-->
+							<!-- wp:group {"align":"full","style":{"spacing":{"blockGap":"0.1em","padding":{"top":"0.1em","right":"0.1em","bottom":"0.1em","left":"0.1em"}},"border":{"width":"1px","color":"#c7c3c34f"}},"layout":{"inherit":false}} -->
```

Comment:
Do we need to explicitly set block gap? It seemed fine when I left it out. For padding, I think it would be better to use fixed units or it could get too large (maybe `10px`?). Just curious how the color value was selected?

Reply:
> Do we need to explicitly set block gap? It seemed fine when I left it out

I actually tried without using the block gap, but they seemed getting a bit too far from each other when in Group block. That's why had to add the gap. But the user can change it from the block settings.

> For padding, I think it would be better to use fixed units or it could get too large (maybe 10px?).

Very good suggestion! Didn't consider this. Updated them to fixed units [here](https://github.com/Automattic/sensei/pull/5576/commits/fa1df378b2057ed0e423505daa605880ccb77aa5), thanks!!

> Just curious how the color value was selected?

I tried to determine the border color from figma for individual course. Didn't find any border there
![Screen Shot 2022-09-08 at 12 03 22 AM](https://user-images.githubusercontent.com/6820724/188960666-adf69a04-2b9c-4af9-a113-2b727604f044.png)
, but block shadow was used which appeared as border. I tried picking the color from the shadow. This is also configurable from the border setting.